test/integration/scheduler_perf/scheduler_perf_types.go

```diff
@@ -0,0 +1,30 @@
+package benchmark
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+		"k8s.io/kubernetes/pkg/api/v1"
+)
+
+// High Level Configuration for all predicates and priorities.
+type schedulerPerfConfig struct {
```

Comment:
so, imo, There isnt much value in adding more cardinality to the nodeAffinity when we're only supporting one filter for now.   i guess we will move forward and make handling the more complex scenarios our next priority for the improvements.